osi_logicallane.proto

```diff
@@ -0,0 +1,502 @@
+//     more data needed. This can mostly be mitigated by only transmitting the
+//     lane boundaries during initialization (e.g. via the OSMP GroundTruthInit
+//     message).
+// - The LogicalLaneBoundary contains all data directly which an agent model is
+//   likely to need. It does not include information normally only used by
+//   sensor models (e.g. the exact length of the color markings on the road).
+//   This information can be gotten from the physical lane referenced in the
+//   LogicalLane, if needed.
+//
+// TODO To Discuss: It would be entirely possible to add ST coordinates also to
+// LaneBoundary. Then no separate type would be needed. However, it is unclear if
+// the additional overhead is acceptable for sensor models.
+//
+message LogicalLaneBoundary
+{
+    // The ID of the lane boundary.
+    //
+    // \rules
+    // is_globally_unique
+    // \endrules
+    //
+    optional Identifier id = 1;
+
+    // points making up the lane boundary.
+    // The boundary must be defined in the same direction as the reference
+    // line. So S positions should increase along the line. Note that S
+    // positions will not always increase strictly monotonically.
+    // Example:
+    //                |---------\
+    //                |          \
+    //  /---boundary--|           \------------
+    // /
+    // ---------- reference line --------------
+    // At the place where the boundary suddenly increases, two points will have
+    // the same S coordinate.
+    //
+    // If the boundary approximates a curve (e.g. a cubic function in
+    // OpenDRIVE), the points must be chosen in a way that the lateral distance
+    // to the ideal line does not exceed 5cm. As shown in the following image:
+    //
+    // \image html line_approximation_error.svg "Approximation error"
+    // Approximation error green line.
+    //
+    // The Z error (difference in Z height between boundary_line and the "real"
+    // line) must also not exceed 5cm.
+    //
+    // Note: if two lanes have different Z heights (e.g. a driving lane is
+    // beside a sidewalk, where the sidewalk is 10cm higher than the road),
+    // then these lanes cannot share a boundary, since their boundaries have
+    // different Z heights.
+    //
+    repeated LogicalBoundaryPoint boundary_line = 2;
+
+    // The reference line for this boundary.
+    //
+    // The reference line is used as a coordinate system for this boundary.
+    // All points of this LogicalLaneBoundary must have S coordinates in the
+    // range [sStart,sEnd].
+    //
+    // The reference line should roughly have the same shape as the boundary, so
+    // that S coordinates continually increase along the lane.
+    //
+    optional Identifier reference_line_id = 3;
+
+    // The classification of the lane boundary.
+    //
+    optional LaneBoundary.Classification classification = 4;
+
+    // Optional external reference to the lane boundary source.
```

Comment:
This LaneBoundary classification is reused from the physical LaneBoundary. i.e noLine, Dashes etc.
As this is a logical lane we could maybe replace this with logical information passing to the next lane is allowed.
NoLine and dashed would in both case be passing. 
But we must state in with direction the passing/or noPassing applies.
While I was thinking on how and where to this comment, I realised, that we must discuss Solid-broken roadmarks in our discussion of physical and logical LaneBoundary. I put something on the miro board we can use as a basis for our discussion.

Reply:
We have to keep in mind that logical lane boundaries might be placed even if there is no "real" LaneBoundary (as maybe on wider bi-directional streets), but i agree that maybe the LaneBoundary classification doesnt have to be used here and be replaced with your idea of passing.

Reply:
Done in a new version of the pull request.